opentracing/example/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+// This is a near wholesale copy of the opentracing-go/examples/dapperish.go
+// file. The only thing that has been changed is the addition of an Appdash
+// store, traceapp, and appdash/opentracing Tracer. Everything else is the
+// exact same to show that it changing backends is O(1).
+// One thing I've noticed is the clash of package names, since there is
+// opentracing/opentracing and appdash/opentracing
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"sourcegraph.com/sourcegraph/appdash"
+	otappdash "sourcegraph.com/sourcegraph/appdash/opentracing"
```

Comment:
I've been thinking about the package name since this is unwieldy, and I was thinking instead of having the package name being `opentracing`, it can be renamed to something in the same vein of `standardtracer`, so maybe `appdashtracer`?